ui/desktop/electron-app/src/cli/path.js

```diff
@@ -19,8 +20,15 @@ const isBuiltInCli = existsSync(builtInCliPath);
 /**
  * Returns Boundary CLI path if the CLI is built in or the Boundary binary name
  * if not, so we assume boundary is available within user $PATH.
+ * For Windows, we filter out the path that is within the current working directory.
  */
-const pathBoundary = isBuiltInCli ? builtInCliPath : binaryName;
+const pathBoundary = isBuiltInCli
+  ? builtInCliPath
+  : isWindows()
+  ? which
+      .sync(binaryName, { nothrow: true, all: true })
+      .filter((binary) => !binary.startsWith(process.cwd()))[0]
```

Comment:
question for understanding: what is line 30 doing, not sure if i understand the condition for filtering out?

Reply:
so we want to filter out the binary path that is in our current directory as that is not correct and we only need the path from the CLI that is installed if there's one packaged within the build. 

`which` command returns all available paths of `boundary` in array format.  

I will explain in the comment a bit more since it is not clear

Reply:
That helps explain it thank you!